### PR TITLE
Implement import from Kukai

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/store/wallet/wallet-actions.ts [hunk 1]
```diff
@@ -15,14 +15,16 @@ export const addHdAccountAction = createAction<AccountInterface>('wallet/ADD-HD-
...
 
-export const loadTokenSuggestionActions =
-  createActions<Pick<TokenMetadataInterface, 'id' | 'address'>, TokenMetadataInterface, string>(
-    'assets/LOAD_TOKEN_SUGGESTION'
-  );
-export const loadTokenMetadataActions =
-  createActions<Pick<TokenMetadataInterface, 'id' | 'address'>, TokenMetadataInterface, string>(
-    'assets/LOAD_TOKEN_METADATA'
-  );
+export const loadTokenSuggestionActions = createActions<
```
Comment (keshan3262) [inline src/store/wallet/wallet-actions.ts:18]: I had to regenerate yarn.lock for a lot of times. It looks like one of lint plugins was updated

File: src/screens/import-account/import-wallet/import-wallet.form.tsx [hunk 2]
```diff
@@ -13,3 +17,29 @@ export const importWalletValidationSchema: SchemaOf<ImportWalletFormValues> = ob
...
+  password: string;
+  shouldUseFilePasswordForExtension?: boolean;
+};
+
+export const importKukaiWalletValidationSchema: SchemaOf<ImportKukaiWalletFormValues> = object().shape({
+  keystoreFile: object().test(
+    'keystore-file',
+    'A keystore file is required',
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    (value: unknown) => typeof value === 'object' && isDefined(value) && isString((value as any).uri)
```
Comment (andrewkryshtal) [inline src/screens/import-account/import-wallet/import-wallet.form.tsx:32]: what documentation says about typisation?

Comment (unixvb) [reply src/screens/import-account/import-wallet/import-wallet.form.tsx:32]: could you set `FileInputValue` as `value` type?

Comment (keshan3262) [reply src/screens/import-account/import-wallet/import-wallet.form.tsx:32]: I haven't looked at documentation
Thanks for paying attention, I can create a typeguard function to reduce the amount of kludges

Comment (keshan3262) [reply src/screens/import-account/import-wallet/import-wallet.form.tsx:32]: Or do something else, I don't know for now

**Review by andrewkryshtal — APPROVED** [review]

**Review by andrewkryshtal — APPROVED** [review]

**Pull request merged**